component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java

```diff
@@ -211,13 +203,16 @@ public void run() {
 
                 @Override
                 public void close() {
+                    log.info("close component manager.");
                     if (!closed.compareAndSet(false, true)) {
+                        log.info("Component manager already closed");
                         return;
                     }
                     try {
                         synchronized (CONTEXTUAL_INSTANCE) {
                             if (CONTEXTUAL_INSTANCE.compareAndSet(this, null)) {
                                 try {
+                                    log.info("Component manager : remove shutdown");
```

Comment:
- set message to `ComponentManager : removing shutdown hook`
- set level to debug

Reply:
done